Content.Server/Shuttles/Commands/FTLDiskBurnerCommand.cs

```diff
@@ -0,0 +1,176 @@
+                        );
+                    continue;
+                }
+
+                // check if our destination works already, if not, make it.
+                if (!_entManager.TryGetComponent<FTLDestinationComponent>(dest, out var ftlDestComp))
+                {
+                    FTLDestinationComponent ftlDest = _entManager.AddComponent<FTLDestinationComponent>(dest);
+                    ftlDest.RequireCoordinateDisk = true;
+
+                    if (_entManager.HasComponent<MapGridComponent>(dest))
+                    {
+                        ftlDest.BeaconsOnly = true;
+
+                        shell.WriteLine(
+                            destinations +
+                            " is a planet map " +
+                            dest +
+                            " and will require an FTL point. It may already exist."
+                            );
+                    }
+                }
+                else
+                {
+                    // we don't do these automatically, since it isn't clear what the correct resolution is. Instead we provide feedback to the user and carry on like they know what theyre doing.
+                    if (ftlDestComp.Enabled == false)
+                        shell.WriteLine(
+                            destinations +
+                            " is on map " +
+                            dest +
+                            " that already has an FTLDestinationComponent, but it is not Enabled! Set this manually for safety.");
+
+                    if (ftlDestComp.BeaconsOnly == true)
+                        shell.WriteLine(
+                            destinations +
+                            " is on map " +
+                            dest +
+                            " that requires a FTL point to travel to! It may already exist."
+                            );
+                }
+
+                // create the FTL disk
+                EntityUid cdUid = _entManager.SpawnEntity(CoordinatesDisk, coords);
+                var cd = _entManager.EnsureComponent<ShuttleDestinationCoordinatesComponent>(cdUid);
+                cd.Destination = dest;
+                _entManager.Dirty(cdUid, cd);
+
+                if (_entManager.TryGetComponent<MetaDataComponent>(dest, out var meta) && meta != null && meta.EntityName != null)
+                {
+                    var labelSystem = _entSystemManager.GetEntitySystem<LabelSystem>();
+                    labelSystem.Label(cdUid, meta.EntityName);
+                }
+            }
+            else
+            {
+                shell.WriteLine(destinations + " is not an EntityID");
+            }
+        }
+    }
+
+    public override CompletionResult GetCompletion(IConsoleShell shell, string[] args)
+    {
+        if (args.Length >= 1)
+            return CompletionResult.FromHintOptions(CompletionHelper.MapUids(_entManager), "Map netId");
```

Comment:
Double-check that this completion works with the current master; it seemed to me like it stopped working after merging master into my test PR branch.

Reply:
The completion works, someone just messed up the helper in engine so it returns empty strings.
https://github.com/space-wizards/RobustToolbox/commit/26b09283f3e8ad60842a1564743f4da491c17c7e#diff-fcc1817dc485c3ac2fc001b91638163a3c8e4b37a75ce4a474840e83028ac44eL147

Reply:
Made an engine PR to fix it, but I wouldn't hold this up on account of that. Its just a help text.
https://github.com/space-wizards/RobustToolbox/pull/5495